_env.platform.zsh

```diff
@@ -21,13 +29,33 @@ export CRED_TOKEN=$(op read "op://Employee/$ONE_PASSWORD_ITEM/CRED_TOKEN")
 export ASTER_URL=${ASTER_LOGIN_URL}
 export ASTER_USER_EMAIL=${ASTER_EMAIL}
 export ASTER_USER_PASSWORD=${ASTER_EMAIL_PWD}
-export FIREBASE_API_KEY=$(op read "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key") 
+
+if FIREBASE_API_KEY=$(op read "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key"); then
+  export FIREBASE_API_KEY
+elif FIREBASE_API_KEY=$(op read "op://Cloud Development/Firebase/LOCAL/Api key"); then
+  echo "Using FIREBASE_API_KEY from op://Cloud Development/Firebase/LOCAL/Api key"
+  export FIREBASE_API_KEY
+else
+  echo "FIREBASE_API_KEY not fond neigher in"
+  echo "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key"
+  echo "nor"
+  echo "op://Cloud Development/Firebase/LOCAL/Api key"
+  echo "Please set FIREBASE_API_KEY"
+  retun 1
+fi
 
 # Tests ####
 export DATAOPS_TEST_EMAIL_PASSWORD=${DATAOPS_TEST_EMAIL_PASSWORD:-UniterestingValue}
 export SDC_START_EXTRA_PARAMS="--stage-lib orchestrator jdbc"
 
-py-4.x-stf
+if command -v py-4.x-stf &> /dev/null; then
+  py-4.x-stf
+elif command -v 4x &> /dev/null; then
```

Comment:
Not a fan of adhering to everyone's preference... Love the check, but would you mind moving to the `_env.prerequistes.zsh` I suggested earlier?